### ISSUE TITLE
Accuracy results on evaluation datasets

### ISSUE THREAD

**Issue by davidpagnon** [JunkyByte/easy_ViTPose#55]
Hi, 

I was searching for accuracy results on evaluation datasets on the official [ViTPose repository](https://github.com/ViTAE-Transformer/ViTPose), and I'm not quite sure which ones correspond to the .onnx models you provide. 

I am mostly interested in results on:
- coco_25: Do you have any benchmark results? I cannot find any
- wholebody: Does it correspond to the ViTPose++ models trained on the `COCO+AIC+MPII+AP10K+APT36K+WholeBody` datasets ([here](https://github.com/ViTAE-Transformer/ViTPose?tab=readme-ov-file#wholebody-dataset))?
- coco: Are we talking about the [single-task training](https://github.com/ViTAE-Transformer/ViTPose?tab=readme-ov-file#results-from-this-repo-on-ms-coco-val-set-single-task-training) (classic or simple decoder?), or [multitask training](https://github.com/ViTAE-Transformer/ViTPose?tab=readme-ov-file#results-from-this-repo-on-ms-coco-val-set-single-task-training)? On which datasets?

Thanks in advance!

**Comment by JunkyByte** [comment]
Hello! The coco 25 is an unofficial ckpt finetuned by me from the original vitpose COCO ckpt using a COCO+feet dataset so it does not have an official counterpart.

The other models are created by using model split on the multitask training ckpts if I remember correctly. there are a few issues regarding this, check #18 #36 #8

Honestly it's been a while I don't remember the details, you have to crawl on old issues / PRs for the full context and references :) 

lmk if you have any further questions!

🚀

**Comment by davidpagnon** [comment]
Thanks for your answers!

So no benchmarked evaluation for the coco_25 model. This one you got by fine-tuning the last layer, but the others were converted rather than retrained, right?

I'm still not sure which ones correspond to yours in the official repository: even in the multitask Coco section, there are 3 different base models with different results: ViTPose-B, ViTPose-B*, ViTPose++-B.

For most other keypoint sets, only the ViTPose++ models are available. Is it safe to assume that you underwent the same convertion process for all models, and that the converted coco model is also the ++ one?

When available, the * models seem to have slightly better results, but I'm not even sure what these 3 variations mean.

Thanks in advance!

**Comment by JunkyByte** [comment]
Yes all converted not retrained.

To verify exactly which models we are using I think the most straightforward way is to download original ckpts from ViTPose repo, apply model_split script and check if the converted model and the ckpts in this repo match.

These days I'm in another country and I cannot assist you further. I can run this check myself in a few days

--

Still I think we are using vitpose++ versions

**Comment by davidpagnon** [comment]
Thank you!
